dataactvalidator/config/sqlrules/b20_award_financial.sql

```diff
@@ -0,0 +1,20 @@
+SELECT af.row_number,
+	af.allocation_transfer_agency,
+	af.agency_identifier,
+	af.beginning_period_of_availa,
+	af.ending_period_of_availabil,
+	af.availability_type_code,
+	af.main_account_code,
+	af.sub_account_code,
+	af.program_activity_code,
+	af.object_class
+FROM award_financial AS af
+WHERE af.submission_id = {}
+	AND NOT EXISTS (
+		SELECT 1
+		FROM object_class_program_activity AS op
+		WHERE af.tas IS DISTINCT FROM op.tas
+			AND af.program_activity_code IS DISTINCT FROM op.program_activity_code
+			AND af.object_class IS DISTINCT FROM op.object_class
+			AND af.submission_id = op.submission_id
```

Comment:
Should this be:
```sql
WHERE af.submission_id = op.submission_id AND (
    af.tas IS DISTINCT FROM op.tas
    OR af.program_activity_code IS DISTINCT FROM op.program_activity_code
    OR af.object_class IS DISTINCT FROM op.object_class
)
```
? Otherwise _all three_ fields have to be different to trigger an error, right?